test/message_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'minitest/autorun'
+require_relative '../lib/twiglet/message'
+
+describe Twiglet::Message do
```

Comment:
This whole file is technically duplicated coverage, but I thought it was worth including anyway? Open to getting rid of it 🤷

Reply:
I think we can now remove the tests in logger which check message exists or some tests along those lines because now its responsibility of message class. But I am okay with having duplicate tests as these tests are not costly at all.

Reply:
I prefer to have explicit coverage of functional classes, even if we do end up with duplicate coverage of dependant classes (upto a point). I don't think it is problematic here.

Reply:
@punitcse I thought about removing some tests but I do think it's still important to test that when we call `log.warn` with an invalid message, we still raise, for example. Especially because the tests are so fast, there is very little cost to run them and a high potential cost to breaking that part of the code.